platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -96,7 +103,8 @@ - (void)startWithCompletionHandler:(MGLMapSnapshotCompletionHandler)completion;
 - (void)startWithQueue:(dispatch_queue_t)queue completionHandler:(MGLMapSnapshotCompletionHandler)completion;
 {
     if ([self isLoading]) {
-        NSDictionary *userInfo = @{NSLocalizedDescriptionKey: @"Already started this snapshotter"};
+        NSString *errorMessage = NSLocalizedStringWithDefaultValue(@"ALREADY_STARTED_SNAPSHOTER", nil, nil, @"Already started this snapshotter", "User-friendly error description");
```

Comment:
NSError is meant to be used for errors that are outside the developer’s control, and its localized error description may be displayed to the end user. But if a snapshotter can’t be reused and the developer tries to reuse one, then that’s avoidable programmer error, and we should assert instead of raising an error. If a snapshotter can be reused, then this error should be removed.

Either way, it does seem useful to retain an error parameter just in case, even if it goes unused for the time being.